### PR TITLE
[Merged by Bors] - style: remove mathport output

### PR DESCRIPTION
**Pull request by Komyyy**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Data/DFinsupp/Basic.lean [hunk 1]
```diff
@@ -1522,7 +1522,6 @@ theorem sigmaCurry_single [∀ i, DecidableEq (α i)] [∀ i j, Zero (δ i j)]
     simp [hi]
 #align dfinsupp.sigma_curry_single DFinsupp.sigmaCurry_single
 
-/- ./././Mathport/Syntax/Translate/Expr.lean:107:6: warning: expanding binder group (i j) -/
```
Comment (YaelDillies) [inline Mathlib/Data/DFinsupp/Basic.lean:1525]: Eg this one is still applicable since there is `Π₀ (i) (j), δ i j` just a few lines under.

**Review by YaelDillies — DISMISSED** [review]
Can you please check that the comments are not applicable anymore?

**Comment by Ruben-VandeVelde** [comment]
I agree with Yaël. Please keep the binder comments

**Comment by Komyyy** [comment]
@YaelDillies @Ruben-VandeVelde Sorry. I've created the issue #13496.
I reverted removing `expanding binder group`s, except `Mathlib.Analysis.Matrix` which is rewritten from `∑ k j, ‖A i j‖₊ * ‖B j k‖₊` to `∑ k, ∑ j, ‖A i j‖₊ * ‖B j k‖₊` during porting.

**Review by YaelDillies — COMMENTED** [review]
Great! Thanks. The remaining changes all look agreeable.

maintainer merge

**Comment by PatrickMassot** [comment]
bors merge

**Pull request closed without merging**